pages/doc/proxies_configuring.md

```diff
@@ -522,10 +522,19 @@ Ex: 0 </td>
 </thead>
 <tbody>
 <tr>
+<a name="traceJaegerHttpListenerPorts"></a> 
+<td>traceJaegerHttpListenerPorts</td>
+<td markdown="span">TCP ports to receive Jaeger Thrift formatted data via HTTP. The data is then sent to Wavefront in the [span format](trace_data_details.html#wavefront-span-format). <br/> Default: None.</td>
```

Comment:
Maybe ... in [Wavefront span format]( ) (no article, but add Wavefront)?

Reply:
Thank you! Updated! :)